dbbackup/management/commands/dbrestore.py

```diff
@@ -77,7 +77,8 @@ def restore_backup(self):
         if not self.filepath:
             self.log("  Finding latest backup", 1)
             filepaths = self.storage.list_directory()
-            filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
+            # TODO: It is a bad filter
+            # filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
```

Comment:
Wondering a bit why it was necessary to remove the filter? I mean, there *could* potentially be other files in the file_path that isn't the backup?